### ISSUE TITLE
Event Hub or IoT hub for IoTs?

### ISSUE THREAD

**Issue by vkpopuri** [MicrosoftDocs/architecture-center#2103]
This feedback is about the documentation provided in link, https://docs.microsoft.com/en-us/azure/architecture/example-scenario/dataplate2e/data-platform-end-to-end#feedback

I am thinking IoT hub is the preferred solution for processing sensor data especially in situations where device management, communication and Azure Edge capabilities are required. Perhaps we should separate the diagram by adding IoT hub for IoT devices and Event Hub for Big data streaming on the source side.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5f10895e-4e1d-1cdd-2d57-32880e0cc45a
* Version Independent ID: 3518d0a5-ffd1-dfa7-6d6b-de2a9ef170d6
* Content: [Azure data platform end-to-end - Azure Example Scenarios](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/dataplate2e/data-platform-end-to-end#feedback)
* Content Source: [docs/example-scenario/dataplate2e/data-platform-end-to-end.md](https://github.com/microsoftdocs/architecture-center/blob/master/docs/example-scenario/dataplate2e/data-platform-end-to-end.md)
* Service: **architecture-center**
* Sub-service: **example-scenario**
* GitHub Login: @fabragaMS
* Microsoft Alias: **pnp**

**Comment by absheik** [comment]
Thanks for reporting - this issue is under review.  This is a Microsoft Internal DevOps Bug ID AB#210882

**Comment by DixitArora-MSFT** [comment]
@vkpopuri Just following up if you have gone through this https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-compare-event-hubs

**Comment by vkpopuri** [comment]
Yes, I just checked it.  It says "To summarize, both solutions are designed for data ingestion at a massive scale. Only IoT Hub provides the rich IoT-specific capabilities that are designed for you to maximize the business value of connecting your IoT devices to the Azure cloud. If your IoT journey is just beginning, starting with IoT Hub to support your data ingestion scenarios will assure that you have instant access to the full-featured IoT capabilities once your business and technical needs require them."
I just made my first comment on the architecture diagram as it is generalizing that Event hub is the only way to process IoT sensor data while in reality, IoT hub might be the preferred solution where certain IoT capabilities are required as mentioned in the above paragraph.
